internal/stubstatshandler/stubstatshandler.go

```diff
@@ -0,0 +1,72 @@
+/*
+ *
+ * Copyright 2023 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package stubstatshandler is a stubbable implementation of
+// google.golang.org/grpc/stats.Handler for testing purposes.
+package stubstatshandler
+
+import (
+	"context"
+
+	"google.golang.org/grpc/stats"
+
+	testgrpc "google.golang.org/grpc/interop/grpc_testing"
+)
+
+// StubStatsHandler is a stats handler that is easy to customize within
+// individual test cases.
+type StubStatsHandler struct {
+	// Guarantees we satisfy this interface; panics if unimplemented methods are
+	// called.
+	testgrpc.TestServiceServer
```

Comment:
Why are you bundling a stats handler with a service implementation?  That seems undesirable unless it's somehow important to keep the two things together.  (And now that I've look at the rest of the code, it seems unused and unnecessary.)

Also, we should put this under `testutils` (either in that package or a subdirectory of it).

Reply:
Ah, whoops, this is leftover from the stubserver copy paste. Changed to embedding a stats.Handler.

Reply:
Moved file to within testutils as internal/testutils/stubstatshandler.go.